SparseGrids/CMakeLists.txt

```diff
@@ -56,7 +56,6 @@ set(Tasmanian_source_libsparsegrid_cuda
        ${CMAKE_CURRENT_SOURCE_DIR}/../InterfaceTPL/tsgCudaWrappers.cpp
        tsgCudaKernels.cu)
 
-
```

Comment:
Why remove the line?

Reply:
I added some modifications there and deleted them, but forgot to add back the newline. It should be fixed now.